CI.hs

```diff
@@ -70,13 +75,13 @@ stackResolverOpt = \case
   Just resolver -> "--resolver " ++ resolver
   Nothing -> ""
 
-ghcFlavorOpt :: String -> GhcFlavor -> String
-ghcFlavorOpt daFlavor = \case
+ghcFlavorOpt :: GhcFlavor -> String
+ghcFlavorOpt = \case
     Ghc8101 -> "--ghc-flavor ghc-8.10.1"
     Ghc881 -> "--ghc-flavor ghc-8.8.1"
     Ghc882 -> "--ghc-flavor ghc-8.8.2"
     Ghc883 -> "--ghc-flavor ghc-8.8.3"
-    Da -> "--ghc-flavor " <> daFlavor
+    Da { flavor } -> "--ghc-flavor " <> flavor
```

Comment:
This is good. That is the flag that gets pushed "downstream" into `Ghclibgen`. I like it.